content/agent/basic_agent_usage/_index.md

```diff
@@ -1,4 +1,95 @@
 ---
-title: Basic Agent Usage
-external_redirect: /agent/basic_agent_usage/amazonlinux/
+title: Basic Agents Usage
+kind: documentation
 ---
+
+{{< partial name="platforms/platforms.html" >}}
+
+## Configuration management tools
+
+Manage the Datadog Agent and [integrations](/integrations) using configuration management tools:
+
+### Chef
+* [Chef Github project](https://github.com/DataDog/chef-datadog)
+* [Chef installation](https://app.datadoghq.com/account/settings#integrations/chef)
+* [Chef documentation](/integrations/chef)
+
+### Puppet
+* [Puppet Github project](https://github.com/DataDog/puppet-datadog-agent)
+* [Puppet installation](https://app.datadoghq.com/account/settings#integrations/puppet)
+* [Puppet documentation](/integrations/puppet)
+
+### Ansible
+* [Ansible Github project](https://github.com/DataDog/ansible-datadog)
+* [Ansible installation](https://app.datadoghq.com/account/settings#agent/ansible)
+* [Ansible documentation](/integrations/ansible/)
+
+Chef, Puppet, and Ansible [integrations](/integrations) use our public APIs, so if you're interested in using another automation tool, the above could be leveraged as examples to get you started.
+
+There is also community support for Saltstack:
+
+* Saltstack Formula - https://github.com/DataDog/datadog-formula
+
+## Configuration file
+
+The configuration file for the Agent is located at:
+
+{{% table responsive="true" %}}
+| OS                                                                       | Agent v5                                                                   |  Agent v6                       |
+| :-------                                                                 | :--------                                                                  | :--------                       |
+| [Mac OS X](/agent/basic_agent_usage/osx)                                 | `~/.datadog-agent/datadog.conf`                                            | `~/.datadog-agent/datadog.yaml` |
+| [Linux](/agent/basic_agent_usage/ubuntu)                                 | `/etc/dd-agent/datadog.conf`                                               |                                 |
+| SmartOS                                                                  | `/opt/local/datadog/agent/datadog.conf`                                    |                                 |
+| [Source](/agent/basic_agent_usage/source)                                | `~/.datadog-agent/agent/datadog.conf`                                      |                                 |
```

Comment:
I'd put `n/a` for A6 here to stress that there's not such thing as a "source install" on A6

Reply:
👌